gallery/src/PageHeader/HostedVersionsSelect.scss

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ 
+.gallery-hosted-versions-select {
+  display: flex;
```

Comment:
Should this really be flex? I'd think we'd want the "Version" text baseline-aligned with the dropdown (which it isn't currently) and could just use whitespace or margin for the space inbetween, rather than needing `gap`.

Reply:
> Should this really be flex? I'd think we'd want the "Version" text baseline-aligned with the dropdown (which it isn't currently) and could just use whitespace or margin for the space inbetween, rather than needing `gap`.

This was cleaned up here: 18aab428bcb0badf4d66b08b695127dd78b9a43e